doc/tutorials/objdetect/aruco_detection/aruco_detection.markdown

```diff
@@ -0,0 +1,802 @@
+
+The simplest case is using the same value for `adaptiveThreshWinSizeMin` and
+ `adaptiveThreshWinSizeMax`, which produces a single thresholding step. However, it is usually better to use a
+ range of values for the window size, although many thresholding steps can also reduce the performance considerably.
+
+Default values:
+
+- `int cv::aruco::DetectorParameters::adaptiveThreshWinSizeMin = 3`
+- `int cv::aruco::DetectorParameters::adaptiveThreshWinSizeMax = 23`
+- `int cv::aruco::DetectorParameters::adaptiveThreshWinSizeStep = 10`
+
+#### adaptiveThreshConstant
+
+The `adaptiveThreshConstant` parameter represents the constant value added in the thresholding operation (see OpenCV
+`threshold()` function for more details). Its default value is a good option in most cases.
+
+Default value:
+
+- `double cv::aruco::DetectorParameters::adaptiveThreshConstant = 7`
+
+
+### Contour filtering
+
+After thresholding, contours are detected. However, not all contours
+are considered as marker candidates. They are filtered out in different steps so that contours that are
+very unlikely to be markers are discarded. The parameters in this section customize
+this filtering process.
+
+It must be noted that in most cases it is a question of balance between detection capacity
+and performance. All the considered contours will be processed in the following stages, which usually have
+a higher computational cost. So, it is preferred to discard invalid candidates in this stage than in the later stages.
+
+On the other hand, if the filtering conditions are too strict, the real marker contours could be discarded and,
+hence, not detected.
+
+#### minMarkerPerimeterRate and maxMarkerPerimeterRate
+
+These parameters determine the minimum and maximum size of a marker, specifically the minimum and maximum marker perimeter. They are not specified in absolute pixel values, instead they are
+specified relative to the maximum dimension of the input image.
+
+For instance, a image with size 640x480 and a minimum relative marker perimeter of 0.05 will lead
+to a minimum marker perimeter of 640x0.05 = 32 pixels, since 640 is the maximum dimension of the
+image. The same applies for the `maxMarkerPerimeterRate` parameter.
+
+If the `minMarkerPerimeterRate` is too low, detection performance can be significantly reduced,
+as many more contours will be considered for future stages.
+This penalization is not so noticeable for the `maxMarkerPerimeterRate` parameter, since there are
+usually many more small contours than big contours.
+A `minMarkerPerimeterRate` value of 0 and a `maxMarkerPerimeterRate` value of 4 (or more) will be
+equivalent to consider all the contours in the image, however this is not recommended for
+performance reasons.
+
+Default values:
+
+- `double cv::aruco::DetectorParameters::minMarkerPerimeterRate = 0.03`
+- `double cv::aruco::DetectorParameters::maxMarkerPerimeterRate = 4.0`
```

Comment:
It's bad idea. Defaults may change in code, but not tutorial. I propose to add reference to API description, but not mention defaults in tutorial text.

Reply:
fixed, added link in `@see` section:

![image](https://github.com/opencv/opencv/assets/22337800/5d4e9221-d706-4990-bc42-6a5333499d94)